projects/kit/components/input-files/input-files.template.html

```diff
@@ -18,14 +18,16 @@
             {{ text }}
         </ng-container>
         <ng-container *ngIf="!readOnly && !computedDisabled">
+            <ng-content select="input[inputFiles]"></ng-content>
             <input
```

Comment:
We should rather switch to native input, if it's present, so that users can listen to native events if they want.